clusterloader2/gcp-csi-driver-stable.yaml

```diff
@@ -0,0 +1,354 @@
+# This config generated from the GCP PD CSI Driver
```

Comment:
I'm not a fan of putting this file in the clusterloader2/ main dir.
Can you move it to "clusterloader2/drivers" or "cl2/csi/" or sth like that.

[e.g. cl2/testing/prometheus contains manifests